Pipfile

```diff
@@ -78,4 +96,6 @@ python_version = "3.6"
 test = "python setup.py test"
 build_sphinx = "python setup.py build_sphinx"
 dev = "pytest --no-cov -vs tests"
+bootstrap = "./scripts/bootstrap"
 setup = "./scripts/setup"
+server = "./scripts/server"
```

Comment:
Shouldn't we adapt bootstrap/server to avoid using `pipenv run` commands into?
And adapt these scripts to check we're in a virtualenv before launching commands?